src/Scrapers/Scraper.py

```diff
@@ -4,22 +4,44 @@
 import requests
 from pathlib import Path
 from ..Config.Config import Config
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+import time
+
 '''
     Scraper abstract class
 '''
 
 class Scraper:
     def __init__(self, url) -> None:
         self.url = url
+    
+    def __getDynamicResponse(self):
```

Comment:
Can you change all of these to snake case.

Reply:
@akstron I have organized the UnsupportedPlatform exception in accordance with that of Runner directory. I have also changed the case to snake case.

Reply:
And as an aside, do you have any idea by when will the ContriHub website be functional again? It currently shows "Application Error" on my side. And it would be great if you/ any other maintainer could review my PR for the memory-game repo, its been there for quite some time.

Reply:
The error will be resolved soon. Also, I have informed the corresponding maintainer of memory-game repo to review your PR.